### DISCUSSION TITLE
tasmota 14.4.1 matter device offline in Google Home.

### DISCUSSION THREAD

**Discussion by Dyrke** [arendst/Tasmota#22731] [Q&A]
Hi I have an esp32s3 running tasmota 14.4.1 and when I deploy it into google home it functions 1 time after that it goes offline and never comes alive again. This behavior started fro m version 14.3.0, and I had hoped it was solved by now. But I still have that problem.
And I can read that there are several others that has had this problem, but some claim it is handled now. Well at least I still have this problem.

**Comment by s-hadinger** [comment]
I'm sorry for your problem. The problem is that it's hard to guess what is your setup and what is your problem. 

Please set `weblog 3` or `seriallog 3` and paste the logs from the pairing time, when it first works, and when it does not work anymore
